src/components/AddToCartButton/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useContext } from 'react';
+import { FaCartPlus } from 'react-icons/fa';
+import CartContext from '../../contexts/cartContext';
+import * as S from './style';
+
+export default function AddToCartButton({ product }) {
+  const { addToCart } = useContext(CartContext);
+
+  return (
+    <S.AddToCartButton onClick={addToCart(product)}>
```

Comment:
Tem que usar arrow function. A função assim tá se auto executando ao carregar e adicionando mais de um produto. 
```suggestion
    <S.AddToCartButton onClick={() => addToCart(product)}>
```

Reply:
sim!! obrigada!

Reply:
commitei sua sugestão, nao sabia que dava pra fazer isso aqui kkk valeu